### ISSUE TITLE
[UI Enhancement] Visual appearance of quota message should not look like an error / Don't show warning for no quota

### ISSUE THREAD

**Issue by nagmat84** [nextcloud/mail#9195]
### Is your feature request related to a problem? Please describe.

Inside the Mail app the icon for the quota information is an exclamation mark. This appears like a warning message for the inexperienced users, even if there is no problem at all. (For example, if only 1% of the quota are already used.

Secondly, if the server does not support quota at all, because the server does not care for quota, the message reads as if something was wrong. Also, if the server reports `0` as the quota limit (which means "unlimited"), the message tells the user that the servers fails to support quotas even though this actually is a good thing for the user.

### Describe the solution you'd like

Please make the message less infringing.

For example, use a little storage icon (e.g. a disc symbol, a "database" symbol) or anything else which does not look like an error symbol unless there is a real warning or error.

In case the server does not support quota at all or reports an unlimited quota do not show any message or at least change the wording to something which does not sound like there was an error.

**Comment by ChristophWurst** [comment]
> ###
> 
> Inside the Mail app the icon for the quota information is an exclamation mark

Mind sharing a screenshot? 

![image](https://github.com/nextcloud/mail/assets/1374172/b62b5a1a-75c2-43f7-8a51-9ea1bc8f69ca)

It's an i.

**Comment by nagmat84** [comment]
> It's an i.

Yes, you are right. My fault. One of my users asked me about the exclamation mark, I only had a glimpse on the icon and did not look thoroughly enough.

However, the general issue still stands. (You could even say, it is worse as the "i" is easily mistaken as an "!"). I would like to suggest two improvements:

 1. The circule around the "i" is much to heavy. Why not using an "i"-icon with a black "i" and without the circle?
 2. Make it a one-line and don't print "quota" in bold. All other items in that menu are a one-liner. This would make it much more consistent with the other items. Separate the current caption "quota" from the message with a colon.
     1. In case there is no quota, just print: `Quota: none` (don't say anything about something not being supported)
     2. In case there is a quota, just print: `Quota: x GB of y GB`
    
    Both is shorted than the current version and doesn't make it so prominent in comparison with the other items in that menu.

**Comment by ChristophWurst** [comment]
The standard material design icon is filled. The vue material design icons have an outline version:

![Bildschirmfoto vom 2023-12-27 17-05-55](https://github.com/nextcloud/mail/assets/1374172/16649c35-1992-4275-9f18-9275a2681ffe)

However, this introduces an inconsistency with the rest of the icons:

![Bildschirmfoto vom 2023-12-27 17-06-26](https://github.com/nextcloud/mail/assets/1374172/cdff7392-4f21-4553-8e6a-4392e16bdbb6)

![image](https://github.com/nextcloud/mail/assets/1374172/073118a2-57ef-4850-bb27-fc969eea1df0)

The plain *i* looks inconsistent, too.

@nimishavijay what do you prefer?

**Comment by nagmat84** [comment]
While I was trying to make my own suggestions, I found out that there is actually an [official symbol for quota, called `icon-quota`,](https://docs.nextcloud.com/server/19/developer_manual/design/icons.html#id69) in the NC developer manual. :-)

Here are two suggestions with the official symbol:

![2a](https://github.com/nextcloud/mail/assets/7428426/da1c572c-e3ff-4e0d-852f-36f2195ad7a1)

![2b](https://github.com/nextcloud/mail/assets/7428426/826369b1-b892-4c2d-b092-5fae3a7ea58e)

But honestly, I believe the most important part is that everything is printed on one line, the bold font is gone and the "not supported" message vanishes.

Here are two suggestions with the first "i" from one of my locally installed fonts which I deemed at least somewhat usable (but still not optimal). Just for the impression:

![1a](https://github.com/nextcloud/mail/assets/7428426/090f7986-e16a-4540-b39a-5cecf5944349)

![1b](https://github.com/nextcloud/mail/assets/7428426/4ae09564-13dd-4b77-9e4f-7a4ae5b2224b)
